### PR TITLE
Add nojnhuh to k8s-staging-cluster-api-azure OWNERS

### PR DESCRIPTION
**Pull request by nojnhuh**
fyi @CecileRobertMichon

### CODE REVIEW

File: registry.k8s.io/images/k8s-staging-cluster-api-azure/OWNERS [hunk 1]
```diff
@@ -11,6 +11,8 @@ reviewers:
 - Jont828
 - jsturtevant
 - nawazkh
+- nojnhuh
+
```
Comment (CecileRobertMichon) [inline registry.k8s.io/images/k8s-staging-cluster-api-azure/OWNERS:15]: extra new line?

Comment (nojnhuh) [reply registry.k8s.io/images/k8s-staging-cluster-api-azure/OWNERS:15]: Not anymore :)

**Review by CecileRobertMichon — COMMENTED** [review]
lgtm

squash?

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**